_posts/2019-11-28-jumpstart.md

```diff
@@ -5,8 +5,13 @@ date: 2019-11-28
 categories:
   - projects
 description: A display to jumpstart your day!
+<<<<<<< HEAD
```

Comment:
It looks like you're committing an incomplete merge conflict resolution, and then cleaning it up in the second commit.